pkg/grafanaclient/api_test.go

```diff
@@ -0,0 +1,212 @@
+package grafanaclient
```

Comment:
1. Please mark it with in `integration` tag
// +build integration
2. Create `test-integration` goal in make file where you: spin up docker container with Grafana and predefined config file; run tests against container; delete container